### DISCUSSION TITLE
Follow the Youtube tutorial but the deploy is failed

### DISCUSSION THREAD

**Discussion by HanCai98** [anuraghazra/github-readme-stats#1931] [Q&A] [answered]
**Describe the bug**
I followed your tutorial on Youtube, but the deploy step on Vercel is failed

**Expected behaviour**
The deploy should work well

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
The error message is shown below
```
The value for maxDuration must be between 1 second and 10 seconds, in order to increase this limit upgrade your plan: https://vercel.com/pricing
```

**Additional context**

**Comment by rickstaa** [comment, marked as answer]
@HanCai98 Thanks for your question. I think the video, since it was released some time ago, does not contain step 6 of the written tutorial (see #1383 for more information). Let me know if that solves your problem. 

> 6: After forking the repo, open the [vercel.json](https://github.com/anuraghazra/github-readme-stats/blob/master/vercel.json#L5) file and change the maxDuration field to 10

**Reply by HanCai98** [reply]
Yes, it works! Thank you!
